defaults/main.yml

```diff
@@ -22,6 +22,7 @@ gitlab_ldap_host: "example.com"
 gitlab_ldap_port: "389"
 gitlab_ldap_uid: "sAMAccountName"
 gitlab_ldap_method: "plain"
+# "simple_tls", "start_tls", "plain", "ssl", "tls"
```

Comment:
If this list is maintained in the GitLab documentation, I'd much rather link to it than have to maintain this list of available options here. Also, this comment should be above the line it is describing.